### ISSUE TITLE
It seems that a 4th level of attributes is failing.

### ISSUE THREAD

**Issue by aichholzer** [danielgtaylor/aglio#146]
Consider this extract:

```
+ Attributes
    + user (required, object)
        + userId: 12345 (required, string)
        + username: aichholzer (required, string)
        + url: http://about.me/aichholzer (required, string)
        + name (object)
            + first: Stefan (required, string)
            + last: Aichholzer (required, string)
            + game (required, object)
                + gameId: 12345 (required, string)
                + days: 5 (required, string)
```

It is being rendered as:

```
{
  "user": {
    "userId": "12345",
    "username": "aichholzer",
    "url": "http://about.me/aichholzer",
    "name": {
      "first": "Stefan",
      "last": "Aichholzer",
      "game": {
        "undefined": null
      }
    }
  }
}
```

Any ideas?

Whatever is deeper than level 4, in this case the `game` property, becomes just `undefined`.

**Comment by pksunkara** [comment]
This might be an issue in drafter.js itself. Need to verify.

**Comment by aichholzer** [comment]
Thank you guys.

**Comment by Techmind** [comment]
Apiary team confirmed this is related to protogonist/drafter.js.
Reported it to apiary team.
Got this:
"Thanks for bug report. We have already fixed this on our parser's master branch. But it is taking us a bit of time to put it into production since we are still working on a major change to the parser."

**Comment by aichholzer** [comment]
:+1: Eagerly waiting for this fix.

**Comment by danielgtaylor** [comment]
@aichholzer @Techmind this is now fixed in Aglio 2.2.0, but there are still some [regression issues](https://github.com/danielgtaylor/aglio/issues/188#issuecomment-153820074) that are worth mentioning. Closing this for the time being, please reopen if you find it is not working!

**Comment by aichholzer** [comment]
@danielgtaylor :+1: 
Thank you.
